src/api/auth.ts

```diff
@@ -407,7 +407,8 @@ export function publishPackage(
                 owners: [ username ],
                 organization: organization ? organization : '',
                 teamPermissions: permissions,
-                memberPermissions: [{ name: username, permission: 'read-write' }]
+                memberPermissions: [{ name: username, permission: 'read-write' }],
```

Comment:
I don't know, maybe we could use only `write: false / true`, because if the user will have a record here, he'll automatically have at least read permissions, so read will always be true... But on the other hand, if we put `read: true` it will be more intuitive so I think we could use your's format... What do you think?